Automation/AppSettings/Templates/AppsettingsDevelopmentTemplate.json

```
@@ -0,0 +1,5 @@
+{
+  "ConnectionStrings": {
+    "Main": "Server=localhost;Database=ClientProject;Trusted_Connection=True;TrustServerCertificate=True;"
+  }
```

Change this to the development server

```suggestion
  "ConnectionStrings": {
    "Main": "Server=dev;Database=ClientProject;Trusted_Connection=True;TrustServerCertificate=True;"
  }
```